pop-api/integration-tests/src/fungibles/mod.rs

```diff
@@ -554,6 +554,6 @@ fn burn_works() {
 		assert_eq!(balance_after_burn, balance_before_burn - amount);
 		// Token is not live, i.e. frozen or being destroyed.
 		assets::start_destroy(&addr, token);
-		assert_eq!(burn(&addr, token, &BOB, amount), Err(Module { index: 52, error: [17, 0] }));
+		assert_eq!(burn(&addr, token, &BOB, 0), Err(Module { index: 52, error: [17, 0] }));
```

Comment:
Can we mint twice the amount above in bobs account and leave this the same instead please.